libbpf-cargo/src/main.rs

```diff
@@ -81,6 +82,23 @@ enum Command {
         /// Path to top level Cargo.toml
         manifest_path: Option<PathBuf>,
     },
+    Make {
+        #[structopt(short, long)]
+        debug: bool,
+        #[structopt(long, parse(from_os_str))]
+        /// Path to top level Cargo.toml
+        manifest_path: Option<PathBuf>,
+        #[structopt(long, parse(from_os_str), default_value = "/bin/clang")]
```

Comment:
if you specify default_value just "clang", would that work? Not all setups will have /bin/clang exactly there...

Reply:
Yeah, it should search PATH. I can do it as a follow up. There's another place where it's /bin/clang